README.md

```diff
@@ -174,3 +174,21 @@ psabpf-ctl table add pipe <ID> <TABLE> action <ACTION> key <KEY> data <DATA> [pr
    - Each key can be written as IPv4 or MAC address, IPv6 is not supported yet. Values can be specified in binary, octal, decimal or hexadecimal format.
 - DATA - list of action arguments. Each argument can be written as IPv4 or MAC address, IPv6 is not supported yet. Arguments can be specified in binary, octal, decimal or hexadecimal format.
 - PRIORITY - value of priority. Higher value means higher priority.
+
+---
+
+Add an element to a value_set:
+```shell
+psabpf-ctl value_set insert pipe ID VALUE_SET_NAME value DATA
+```
+Delete an element from a value_set:
+```shell
+psabpf-ctl value_set delete pipe ID VALUE_SET_NAME value DATA
+```
+List all elements in the value_set:
+```shell
+psabpf-ctl value_set get pipe ID VALUE_SET_NAME
+```
+- ID - ID of the pipeline, natural number.
+- VALUE_SET_NAME - name of the value_set.
+- DATA - the value_set value. Values can be specified in binary, octal, decimal or hexadecimal format.
```

Comment:
In `README.md` only the most important commands are described, in my opinion parser value_set is not one of these commands. But this command should be added to the [command reference](https://github.com/P4-Research/psabpf/blob/master/docs/command%20reference.md).